broker-cli/commands/wallet.spec.js

```diff
@@ -973,5 +978,12 @@ describe('cli wallet', () => {
       expect(table).to.include(transactions[0].blockHeight)
       expect(table).to.include(transactions[0].pending)
     })
+
+    it('exports transactions as json if option is specified', async () => {
+      reverts.push(program.__set__('JSON', jsonStub))
+      opts.json = true
+      await history(args, opts, logger)
```

Comment:
Do we want to also test that `tablePushStub` has not been called, similar to the test in order summary?